src/debugger/debug_session.ts

```diff
@@ -236,6 +240,7 @@ export class GodotDebugSession extends LoggingDebugSession {
 			args.launch_scene,
 			args.scene_file,
```

Comment:
Could be more concise with
```javascript
get_configuration("scene_file_config", "") || args.scene_file
```
without the `if` and replacing the contents of args.scene_file.

Reply:
That's pretty cool! Will do. Unfortunately have to sleep for now.